platform/core/src/services/PanelService/PanelService.tsx

```diff
@@ -156,6 +156,8 @@ export default class PanelService extends PubSubService {
     options: {
       rightPanelClosed?: boolean;
       leftPanelClosed?: boolean;
+      rightPanelResizable?: boolean;
```

Comment:
I think we made a mistake by tying the panelService to the left and right panels. PanelService is a core service and shouldn't be concerned with location (left, right); it should also support the bottom panel for instance if a layout has it. I'm unsure of the best way to proceed, just sharing my thoughts. We can merge this for now i guess

Reply:
Well these do not need to be explicitly declared here so I will remove them from this file as a first step and maybe that might be enough to create that separation. In essence options can simply be a bag of properties for the various panels, no?